### PR TITLE
fix: CU-UP counters being per UE

### PR DESCRIPTION
**Pull request by ben-foxmoore**
Certain UE counters displayed on the CU-UP PM Counters page are displayed as per UE, whereas some are per CU-UP. This PR changes the ones that are per UE to be per CU-UP, so that an overall view of the CU-UP can be observed.

### CODE REVIEW

**Comment by ben-foxmoore** [comment]
The linter is failing here on me using "CU-UP" at the start of the commit message. Personally, I don't see a problem with that, so I'm tempted to just ignore the check. @Wielewout thoughts on this?

**Comment by Wielewout** [comment]
> The linter is failing here on me using "CU-UP" at the start of the commit message. Personally, I don't see a problem with that, so I'm tempted to just ignore the check. @Wielewout thoughts on this?

The idea is that there should always be some sort of "command" as the first word of a commit expressing what action has been taken with the commit (e.g. `add`, `rename`, `remove`,...). With that there will never be a need for a short-form notation potentially with capitals in the beginning of a commit message.

So for example with this commit I would reword it for example to:

```
fix: organize CU-UP counters per UE
```

**Review by dimgogos — APPROVED** [review]

**Comment by ben-foxmoore** [comment]
Hmm.. I already considered the `fix` to be that action - e.g. "this commit fixes the CU-UP counters being per UE", describing the previous bug, rather than explicitly the change that has been taken.

I think I see some sense in the your approach, although it probably then needs a further sentence or two to describe the problem being fixed (not that this is a bad thing!)

**Comment by Wielewout** [comment]
> Hmm.. I already considered the `fix` to be that action - e.g. "this commit fixes the CU-UP counters being per UE", describing the previous bug, rather than explicitly the change that has been taken.
> 
> I think I see some sense in the your approach, although it probably then needs a further sentence or two to describe the problem being fixed (not that this is a bad thing!)

Fix would also be a valid "command", but it should be repeated then like `fix: fix CU-UP counters being per UE`.

There is a difference between the message and indication of the commit type in the sense that for the release notes the `fix:` just puts it underneath the section of fixes. But the commit message is injected on it's own. So to have a bit of a proper sentence structure for the release notes there needs to be a verb in there which would be that "command".

**Comment by ben-foxmoore** [comment]
> > Hmm.. I already considered the `fix` to be that action - e.g. "this commit fixes the CU-UP counters being per UE", describing the previous bug, rather than explicitly the change that has been taken.
> > I think I see some sense in the your approach, although it probably then needs a further sentence or two to describe the problem being fixed (not that this is a bad thing!)
> 
> Fix would also be a valid "command", but it should be repeated then like `fix: fix CU-UP counters being per UE`.
> 
> There is a difference between the message and indication of the commit type in the sense that for the release notes the `fix:` just puts it underneath the section of fixes. But the commit message is injected on it's own. So to have a bit of a proper sentence structure for the release notes there needs to be a verb in there which would be that "command".

Strictly, that doesn't seem to be what the conventional commits spec expects, at least not always:
> A description MUST immediately follow the type/scope prefix. The description is a short description of the code changes, e.g., fix: array parsing issue when multiple spaces were contained in string.

https://www.conventionalcommits.org/en/v1.0.0/#specification
